esp/esdllib/README.md

```diff
@@ -319,224 +328,317 @@ Some script operations will add additional elements to the context document.
-
-    - The interface of the service
-    - Which methods in the service are enabled
-    - ESDL scripts customizing behavior of the service and its methods
-    - XSLT script customizing the behavior of the service's LogAgents
-    - Configuration of some types of LogAgents
-
-The ```esdl``` command-line tool is the most capable and recommended way to set a service's dynamic configuration. This section discusses the dynamic configuration particularly as it pertains to ESDL Scripting. The dynamic configuration is an xml document with this format:
-
-```xml
-<Binding>
-    <Definition>
-        <LogTransforms/>
-        <LogManager>
-        </LogManager>
-        <Methods>
-            <Scripts>
-                <![CDATA[
-                    <Transforms>
-                        ...
-                    </Transforms>
-                ]]>
-            </Scripts>
-            <Method>
-                <Scripts>
-                    <![CDATA[
-                        <Transforms>
-                            ...
-                        </Transforms>
-                    ]]>
-                </Scripts>
-            </Method>
-        </Methods>
-    </Definition>
-</Binding>
-```
+In order to understand how ESDL Scripts are associated with an ESDL service we'll look at an overview of how services are configured. This is not an exhausive reference, but shows where ESDL Scripts are in a configuration and gives some context of the other main configuration sections. These are our best practice recommendations for configuration. Other approaches are possible, but they aren't covered here.
 
-Each `<Transforms>` element contains ESDL script entry points to execute. Scripts in the `<Methods>` section are referred to as ''service level scripts''. They are shared by each method in a service. Scritps in each `<Method>` section are referred to as ''method-level scripts'' and they are only executed when that method is run. As a the ESP executes a Method request and reaches each script entry point, service-level scripts executed first, in order, followed by the method-level scripts in order. The CDATA section wrapping the `<Transforms>` element ensures that correct script order is preserved.
+An ESDL service is configured from two sources- the *process configuration* and the *service bundle configuration*.
 
-Forthcoming enhancements to the ```esdl``` tool will assemble the `<Binding>` XML, including proper wrapping of CDATA sections, from individual XML script files in your source repo.
+The *process configuration* controls settings such as:
 
-### Using esdl application
+- An ESP process for the service to run inside of
+- Which authentication method and plugin is used for the service
+- TLS settings
+- Trace log settings including transaction summary lines
+- Dali server settings
+- May contain service-level authentication and authorization *Feature* and *Setting* entries
```

Comment:
I agree with this comment and the next for line #351- it may be the result of some incomplete editing. I'll update the wording in this section so it is correct.